### PR TITLE
add test case TestValidateServiceNodePort for validateServiceNodePort method

### PR DESCRIPTION
**Pull request by twilight0620**
#### What type of PR is this?
add test case TestValidateServiceNodePort for validateServiceNodePort method

Add one of the following kinds:
/kind cleanup

#### What this PR does / why we need it:
validateServiceNodePort method does not has ut test, so i add it.

#### Which issue(s) this PR fixes:
add ut test 

#### Does this PR introduce a user-facing change?
```release-note
  NONE

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

<!--

```docs
  NONE

```

### CODE REVIEW

**Comment by twilight0620** [comment]
could anyone review this pr？

**Comment by twilight0620** [comment]
could anyone review this pr？it is still not-ok-to-test

**Comment by aojea** [comment]
/ok-to-test
thanks

File: cmd/kube-apiserver/app/options/validation_test.go [hunk 1]
```diff
@@ -124,6 +125,65 @@ func getIPnetFromCIDR(cidr string) *net.IPNet {
 	return ipnet
 }
 
+func TestValidateServiceNodePort(t *testing.T) {
```
Comment (aojea) [inline cmd/kube-apiserver/app/options/validation_test.go:128]: what happens if size is negative or greater than the port space (65535)?

Comment (aojea) [reply cmd/kube-apiserver/app/options/validation_test.go:128]: ah, ok, the port range is not validated here :)

**Pull request merged**